tests/reader_test.py

```diff
@@ -0,0 +1,64 @@
+import os
+from datetime import datetime
+import unittest
+
+from ibodata.readers import FileReader
+
+import numpy as np
+
+
+class TestReaderRegularData(unittest.TestCase):
+    def setUp(self):
+        """
+        source: path to directory with test files
+        """
+        self.source = os.path.join(os.path.dirname(os.path.abspath(__file__)), "test_files", "2010-11-12_13_14_15")
```

Comment:
this line may be an issue, see for reference:
https://github.com/DataMedSci/pymchelper/blob/master/tests/test_converters.py#L103

I am not sure if `__file__` will work when tests are ran via tox and pytest modules.

Can you try using simply a relative directory ?